doc/classes/TileSetEditor.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TileSetEditor" inherits="HSplitContainer" category="Core" version="3.2">
+	<brief_description>
+		Editor for tilesets.
+	</brief_description>
+	<description>
+		An editor for creating and editing tilesets, including using autotile. Most of the functionality of this class is accessable through [TileSetEditorPlugin].
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+    <method name="edit">
+			<return type="void">
+			</return>
+      <argument index="0" name="p_tileset" type="TileSet">
+      </argument>
+			<description>
+        Sets the [code]p_tileset[/code] to the [TileSet] to be edited.
+			</description>
+		</method>
+    <method name="update_library_file">
```

Comment:
This method is not bound in `TileSetEditor::_bind_methods`, which means it is impossible to call it from GDScript. I think `TileSetEditor` itself is hard to access, due to it not being registered in `ClassDB`.